api/diagnosis/diagnosis_routes_v2.py

```
@@ -0,0 +1,87 @@
+        is_drowsiness_drive = True
+        for img in input_array:  # img.shape == (145, 145, 3)
+            input_tensor = np.expand_dims(img, axis=0)  # shape == (1, 145, 145, 3)
+
+            predicted_class  = model.predict(input_tensor)
+
+            if predicted_class >= 0.00016:
+                is_drowsiness_drive = False
+                break
+
+        detection_time = datetime.now(timezone.utc).isoformat()
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)